projects/plugins/jetpack/modules/contact-form/admin.php

```diff
@@ -1192,7 +1194,7 @@ public static function maybe_enable_gdrive_export_button() {
 			return;
 		}
 
-		$user_connected = ( new Connection_Manager( 'jetpack' ) )->is_user_connected( get_current_user_id() );
+		$user_connected = ( defined( 'IS_WPCOM' ) && IS_WPCOM ) || ( new Connection_Manager( 'jetpack' ) )->is_user_connected( get_current_user_id() );
```

Comment:
Hmm, why this check? Do we always want to show the button on WP.com regardless of `JETPACK_BETA_BLOCKS` or did you just use the wrong constant by accident?

Reply:
p1669932430568779-slack-CDLH4C1UZ

Reply:
The action hooks are set only when Jetpack BETA flag is enabled.